core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-or-update-solution-modal.component.html

```diff
@@ -16,7 +16,8 @@ <h3>{{ solutionType !== null ? 'Update Solution' : 'Add Solution' }}</h3>
       <br>
       <div>
         <oppia-interaction-display classStr="e2e-test-interaction-html"
-                                   [htmlData]="correctAnswerEditorHtml">
+                                   [htmlData]="correctAnswerEditorHtml"
+                                   [parentScope]="this">
```

Comment:
?

Reply:
What? 
It’s the same way that lastAnswer is passed in supplemental card directive.

Reply:
this is not an explanation of doing things, `[parentScope]="this"   ?`

Reply:
Ok, you could have asked the question better then. 

I'm still not exactly sure what you are asking, since this is a normal way of passing parent scope to child directive for bound variables (eg, as I mentioned before, in [supplemental card](https://github.com/oppia/oppia/blob/3628528c259cbf9ada8b850db5d1eb0e4536c8d5/core/templates/pages/exploration-player-page/learner-experience/supplemental-card.component.html#LL33C39-L33C39, where lastAnswer was passed to interaction-display in this same way)).

Is there anything specific you want to know?

Reply:
@aks681 I think this is totally my fault my message is unclear,
Though I have tried to highlight the code and you have added only one single line in the code. and question mark represent why commonly, so I assumed this would make you clear but I am pointing out in the code, I think I have expected more.

Completely sorry, if this takes your time. it's completely my fault.

As per as my knowledge we use observables/EventEmitters/input/output instead of passing this(scope)  to child components. is there some specific reason for passing `this`? 

I don't think this completely justifies your answer that we are using this in the supplemental card, so we are using it here, there is always a reason for doing things I guess, hoping you will understand this.

[Now I have taken a look at the code myself and this is clear to me, why you have added this line in the code].

Thank you

(from the next point onwards I will take care that my messages are pretty much clear).